infrastructure/main.tf

```diff
@@ -58,10 +64,16 @@ module "repl-docmosis-backend" {
     ETHOS_REPL_DB_USER_NAME            = module.db.user_name
     ETHOS_REPL_DB_NAME                 = module.db.postgresql_database
     ETHOS_REPL_DB_CONN_OPTIONS         = local.db_connection_options
+    ETHOS_REPL_DB_HOST_V11             = module.db-v11.host_name
```

Comment:
Although the cnp-module-webapp is probably no longer required we shouldn't take the risk of changing the database credentials - what happens if this is still being used in prod?